watch-deployment.test.js

```diff
@@ -2,9 +2,16 @@ const { watchDeployment } = require("./watch-deployment.js");
 
 const axios = require("axios");
 const MockAdapter = require("axios-mock-adapter");
+const { mockProcessStdout } = require("jest-mock-process");
 
 const mockHarnessApiUrl = "http://harness.com/deployment-api-endpoint";
 const mockHarnessUiUrl = "http://harness.com/watch-deployment";
+var capturedStdout = mockProcessStdout();
+
+// reset the captured calls before each test
+beforeEach(async () => {
+  capturedStdout = mockProcessStdout();
```

Comment:
I'm being eager and setting up capturing for every test, regardless of if it needs it or not. Why? Because it also hides the output, making the test run results much less noisy